packages/rollup-plugin-import-meta-assets/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @web/rollup-plugin-import-meta-assets
 
+## 2.0.3
```

Comment:
Technically a minor version according to semver, right?

Reply:
actually I agree, it's kinda a new feature, so should be 2.1.0 which in this case is a `minor`

@Soesah can you please fix the changelog file accordingly? you can manually update `patch` to `minor` in it without regenerating the file